src/screens/FeedScreen/RenderList.js

```diff
@@ -160,6 +209,17 @@ const RenderListFeed = (props) => {
             hasComment={hasComment}
           />
         )}
+        {isBlurred && (
+          <TopicsChip
+            onLayout={calculateLineTopicChip}
+            topics={item?.topics}
+            fontSize={normalizeFontSizeByWidth(14)}
+            text={item.message}
+            topicContainer={{
+              bottom: topicBottomPosition()
+            }}
+          />
+        )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Ensure that the `calculateLineTopicChip` function and `topicBottomPosition()` method handle potential null or undefined values correctly to avoid runtime errors.



<!-- This is an auto-generated comment by OSS CodeRabbit -->